VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -307,7 +307,14 @@ final public class AuthenticationController
             }
         }
         
-        self.client.authenticatedAccount = nil
+        if loadClientCredentials
+        {
+            self.client.authenticatedAccount = (try? self.accountStore.loadAccount(.ClientCredentials)) ?? nil
```

Comment:
Yessir, I'll add this functionality directly to `AccountStore`, for anytime a fetch errors